### ISSUE TITLE
Rails getting started page uses inexisting gem

### ISSUE THREAD

**Issue by kamaradclimber** [Pictogrammers/pictogrammers.com#94]
### Before You Begin

- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Following https://pictogrammers.com/docs/library/mdi/getting-started/ruby-on-rails/
`material_design_icons` ruby gem does not exist on rubygems.org.

`gem install material_design_icons` fails with:

> ERROR:  Could not find a valid gem 'material_design_icons' (>= 0) in any repository

### Describe the behavior you expected

It should install the gem

### Steps to reproduce the issue

1. Run `gem install material_design_icons`
2. Observe failure

### Additional information

This instruction allows for anyone to start doing squatting to make user install anything when following official documentation

**Comment by Chew** [comment]
Looks like it got yanked: https://rubygems.org/gems/material_design_icons

Curious to see why.

**Comment by Templarian** [comment]
So we don't have a first party solution for this. Is it normal for gems to disappear? Use to a few package managers, but most don't allow deleting only deprecation.

I don't know anything about ruby though, so open to suggestions for what the docs should say going forward??? @kamaradclimber @Chew.

**Comment by Chew** [comment]
Yeah, it's quite odd. Wish we could get an explanation. But, I ended up using [Iconify](https://iconify.design/), which worked for me once I gave it a margin-bottom of 3px to center it a bit. No real need to use a gem anymore in my opinion.
